deeprank/learn/DataSet.py

```diff
@@ -601,7 +601,7 @@ def _get_target_ordering(self):
 
         lower_list = ['IRMSD','LRMSD','HADDOCK']
         higher_list = ['DOCKQ','Fnat']
-        NA_list = ['binary_class','BIN_CLASS']
+        NA_list = ['binary_class','BIN_CLASS', 'class']
```

Comment:
why do you need a new one there ? And maybe a more self describing name would be good

Reply:
What I think is to give a general target name for classification.

Reply:
Remove it then, not that necessary.